### PR TITLE
Update all readmes

### PR DESCRIPTION
**Pull request by merelcht**
## Motivation and Context
Closes https://github.com/kedro-org/kedro/issues/3111

I made all spaceflight starter README.md files consistent. And updated any outdated info.

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Assigned myself to the PR
- [ ] Added tests to cover my changes

### CODE REVIEW

File: spaceflights-pyspark-viz/{{ cookiecutter.repo_name }}/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This is your new Kedro project, which was generated using `kedro {{ cookiecutter.kedro_version }}`.
+This is your new Kedro project for the [spaceflights tutorial](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) and the extra tutorial sections on [visualisation with Kedro-Viz](https://docs.kedro.org/en/stable/visualisation/index.html) and [experiment tracking with Kedro-Viz](https://docs.kedro.org/en/stable/experiment_tracking/index.html) with PySpark setup, which was generated using `kedro {{ cookiecutter.kedro_version }}`.
```
Comment (stichbury) [inline spaceflights-pyspark-viz/{{ cookiecutter.repo_name }}/README.md:5]: Link to subprojects docs -- I can do this

**Review by stichbury — COMMENTED** [review]
I made a commit to this branch to change the READMEs. I removed the content about the example code added by tools for now as we discussed it in our docs meeting. It will come back in some form but we decided we need good separation between starters and the code added by tools, so I need to write this/see it written and then add a sentence or two to each starter where needed to explain. Or maybe just link to the framework docs. So this is still to come but you can do that in a separate PR if you want to merge this one.

File: spaceflights-pandas-viz/README.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ kedro new --starter=spaceflights-pandas-viz
...
 ```
 
-This will give you the complete project and project template. If you would prefer to have a reduced project template you can use `add-ons` instead and select `Kedro-Viz` as add-on with an example:
-```bash
-pip install kedro
-kedro new --add-ons=XXX
-cd <my-project-name>  # change directory into newly created project directory
-```
+Alternatively, you can follow the interactive `kedro new` flow to pick and choose which tools you want as part of your project and choose to add an example to get Spaceflights based example pipelines.
+Depending on your tool selection you will get an example from either of the four spaceflights starters:
```
[the remark this answers is not in the document]
Comment (stichbury) [reply spaceflights-pandas-viz/README.md]: You can ignore this now and we will put it in the docs.

File: spaceflights-pyspark-viz/README.md [hunk 3]
```diff
@@ -2,16 +2,20 @@
 
 ## Overview
 
-This is a variation of the [spaceflights tutorial project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) described in the [online Kedro documentation](https://docs.kedro.org) with `PySpark` and `Kedro-Viz` setup.
+This is a completed version of the [spaceflights tutorial project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) and the extra tutorial sections on [visualisation with Kedro-Viz](https://docs.kedro.org/en/stable/visualisation/index.html) and [experiment tracking with Kedro-Viz](https://docs.kedro.org/en/stable/experiment_tracking/index.html) with PySpark setup that originates from the [Kedro documentation about how to work with PySpark](https://docs.kedro.org/en/stable/integrations/pyspark_integration.html).
```
Comment (stichbury) [inline spaceflights-pyspark-viz/README.md:5]:
```suggestion
This is a completed version of the [spaceflights tutorial project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) and the extra tutorial sections on [visualisation with Kedro-Viz](https://docs.kedro.org/projects/kedro-viz/en/stable/kedro-viz_visualisation.html) and [experiment tracking with Kedro-Viz](https://docs.kedro.org/projects/kedro-viz/en/stable/experiment_tracking.html) with a PySpark setup that originates from the [Kedro documentation about how to work with PySpark](https://docs.kedro.org/en/stable/integrations/pyspark_integration.html). 
```

File: spaceflights-pyspark-viz/README.md [hunk 4]
```diff
@@ -2,16 +2,20 @@
 
 ## Overview
 
-This is a variation of the [spaceflights tutorial project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) described in the [online Kedro documentation](https://docs.kedro.org) with `PySpark` and `Kedro-Viz` setup.
+This is a completed version of the [spaceflights tutorial project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) and the extra tutorial sections on [visualisation with Kedro-Viz](https://docs.kedro.org/projects/kedro-viz/en/stable/kedro-viz_visualisation.html) and [experiment tracking with Kedro-Viz](https://docs.kedro.org/projects/kedro-viz/en/stable/experiment_tracking.html) with a PySpark setup that originates from the [Kedro documentation about how to work with PySpark](https://docs.kedro.org/en/stable/integrations/pyspark_integration.html).
```
Comment (stichbury) [inline spaceflights-pyspark-viz/README.md:5]:
```suggestion
This is a completed version of the [spaceflights tutorial project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) and the extra tutorial sections on [visualisation with Kedro-Viz](https://docs.kedro.org/projects/kedro-viz/en/stable/kedro-viz_visualisation.html) and [experiment tracking with Kedro-Viz](https://docs.kedro.org/projects/kedro-viz/en/stable/experiment_tracking.html) with a PySpark setup that originates from the [Kedro documentation about how to work with PySpark](https://docs.kedro.org/en/stable/integrations/pyspark_integration.html). 
```

**Review by stichbury — APPROVED** [review]
LGTM! Sorry for all the extra commits -- I've fixed links and removed a few bits of content and am happy with the result. See what you think. 🏆

File: spaceflights-pandas/{{ cookiecutter.repo_name }}/README.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ kedro run
 
 ## How to test your Kedro project
 
-Have a look at the file `src/tests/test_run.py` for instructions on how to write your tests. You can run your tests as follows:
+Have a look at the files `src/tests/test_run.py` and `src/tests/pipelines/test_data_science.py` for instructions on how to write your tests. You can run your tests as follows:
```
Comment (merelcht) [inline spaceflights-pandas/{{ cookiecutter.repo_name }}/README.md:38]:
```suggestion
Have a look at the files `src/tests/test_run.py` and `src/tests/pipelines/test_data_science.py` for instructions on how to write your tests. Run the tests as follows:
```

File: spaceflights-pyspark-viz/{{ cookiecutter.repo_name }}/README.md [hunk 6]
```diff
@@ -32,3 +32,67 @@ You can run your Kedro project with:
 ```
 kedro run
 ```
+
+## How to test your Kedro project
+
+Have a look at the files `src/tests/test_run.py` and `src/tests/pipelines/test_data_science.py` for instructions on how to write your tests. You can run your tests as follows:
```
Comment (merelcht) [inline spaceflights-pyspark-viz/{{ cookiecutter.repo_name }}/README.md:38]:
```suggestion
Have a look at the files `src/tests/test_run.py` and `src/tests/pipelines/test_data_science.py` for instructions on how to write your tests. Run the tests as follows:
```

File: spaceflights-pyspark/{{ cookiecutter.repo_name }}/README.md [hunk 7]
```diff
@@ -32,3 +32,67 @@ You can run your Kedro project with:
 ```
 kedro run
 ```
+
+## How to test your Kedro project
+
+Have a look at the files `src/tests/test_run.py` and `src/tests/pipelines/test_data_science.py` for instructions on how to write your tests. You can run your tests as follows:
```
Comment (merelcht) [inline spaceflights-pyspark/{{ cookiecutter.repo_name }}/README.md:38]:
```suggestion
Have a look at the files `src/tests/test_run.py` and `src/tests/pipelines/test_data_science.py` for instructions on how to write your tests. Run the tests as follows:
```

File: spaceflights-pyspark/{{ cookiecutter.repo_name }}/README.md [hunk 8]
```diff
@@ -32,3 +32,67 @@ You can run your Kedro project with:
...
+
+```
+pytest
+```
+
+To configure the coverage threshold, look at the `.coveragerc` file.
+
+## Project dependencies
+
+To see and update the dependency requirements for your project use `requirements.txt`. You can install the project requirements with `pip install -r requirements.txt`.
```
Comment (merelcht) [inline spaceflights-pyspark/{{ cookiecutter.repo_name }}/README.md:48]:
```suggestion
To see and update the dependency requirements for your project use `requirements.txt`. Install the project requirements with `pip install -r requirements.txt`.
```

**Comment by stichbury** [comment]
Thanks for the changes @merelcht -- all look good to me. I committed them.

**Review by noklam — APPROVED** [review]

**Pull request merged**